src/main/java/com/example/rets_api/controller/RetsController.java

```diff
@@ -74,6 +75,19 @@ public ResponseEntity<PropertyDTO> deletePropertiesById(@PathVariable("id") Long
         return handleResponse(propertyDTO);
     }
 
+    @ApiOperation(value = "Patch basic fields in a property by id")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Property patched", response = Long.class ),
+            @ApiResponse(code = 404, message = "Property not founded"),
+            @ApiResponse(code = 500, message = "Internal error")
+    })
+    @PatchMapping(value="/{id}")
+    public ResponseEntity<PropertyDTO> patchProperty(@PathVariable("id") Long propertyId, @RequestBody PropertyPatchDTO propertyPatchDTO){
```

Comment:
I think it's better to send back just the PropertyPatchDTO, because will contain the changes requested. What do you think?